enclave/src/sidechain/mod.rs

```diff
@@ -0,0 +1 @@
+pub mod validateer;
```

Comment:
I'm curious about the naming here: A 'validateer' is something that validates (validater)?

Reply:
Yes, @brenzi introduced that term lately. I am actually not certain if he meant that seriously. It should be a pun from validator + TEE

Reply:
ah okay 😄  - maybe we should write a short comment somewhere to clarify its meaning

Reply:
Absolutely, I'd suppose that the term will reach the general integritee docs anyhow.

Reply:
I meant it seriously. And yes, the term should be introduced somewhere in the code docs